Fix binsize CI

It is failing on main:

https://github.com/unicode-org/icu4x/actions/runs/19732992427/job/56538227332

```
   Compiling icu v2.1.1 (/home/runner/work/icu4x/icu4x/components/icu)
warning: unused variable: `block`
  --> components/icu/examples/unicode_bmp_blocks_selector.rs:68:13
   |
68 |         let block = selector.select(ch);
   |             ^^^^^ help: if this is intentional, prefix it with an underscore: `_block`
   |
   = note: `#[warn(unused_variables)]` (part of `#[warn(unused)]`) on by default

error[E0433]: failed to resolve: use of unresolved module or unlinked crate `writeable`
  --> components/segmenter/examples/experimental_segmenter.rs:37:5
   |
37 |     println!("{} repetitions done in: {:?}", REPETITIONS, elapsed);
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ use of unresolved module or unlinked crate `writeable`
   |
   = help: if you wanted to use a crate named `writeable`, use `cargo add writeable` to add it to your `Cargo.toml`
   = note: this error originates in the macro `println` (in Nightly builds, run with -Z macro-backtrace for more info)

```

--- a/components/segmenter/examples/experimental_segmenter.rs
+++ b/components/segmenter/examples/experimental_segmenter.rs
@@ -4,7 +4,6 @@
 
 #![no_main] // https://github.com/unicode-org/icu4x/issues/395
 icu_benchmark_macros::instrument!();
-use icu_benchmark_macros::println;
 
 #[path = "../tests/adaboost/main.rs"]
 mod adaboost;
@@ -33,7 +32,7 @@ fn main_adaboost(args: &[String]) {
         print!("{}|", &s[prev..breakpoint]);
         prev = breakpoint;
     }
-    println!("");
+    println!();
     println!("{} repetitions done in: {:?}", REPETITIONS, elapsed);
 }
 
@@ -67,7 +66,7 @@ fn main_cnn(args: &[String]) {
         print!("{}|", &s[prev..breakpoint]);
         prev = breakpoint;
     }
-    println!("");
+    println!();
     println!("{} repetitions done in: {:?}", REPETITIONS, elapsed);
 }
 
@@ -94,7 +93,7 @@ fn run_word_segmenter(segmenter: WordSegmenterBorrowed, s: &str) {
         print!("{}|", &s[prev..breakpoint]);
         prev = breakpoint;
     }
-    println!("");
+    println!();
     println!("{} repetitions done in: {:?}", REPETITIONS, elapsed);
 }
 

